.pre-commit-config.yaml

```diff
@@ -1,16 +1,25 @@
 repos:
-  - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.237
-    hooks:
-      - id: ruff
-        args: [--fix]
-        exclude: ^resources
-
   - repo: https://github.com/abravalheri/validate-pyproject
     rev: v0.10.1
     hooks:
       - id: validate-pyproject
 
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v3.0.0-alpha.4
+    hooks:
+      - id: prettier
+        exclude: ^ruff.schema.json$
+
+  - repo: https://github.com/igorshubovych/markdownlint-cli
+    rev: v0.33.0
+    hooks:
+      - id: markdownlint-fix
+        args:
+          - --disable
+          - MD013
+          - MD033
+          - --
```

Comment:
Should this now be removed? (Are we using Prettier instead?)

Reply:
Prettier is only a formatter, it will not catch some lint that markdownlint has.
e.g. [MD001](https://github.com/DavidAnson/markdownlint/blob/main/doc/md001.md) heading-increment/header-increment - Heading levels should only increment by one level at a time

Reply:
We should keep both.

Reply:
I see, does markdownlint do any autofixing? Would the tools ever conflict?